<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <title>api-test-report-20250809-205712.html</title>
    <style>body {
  font-family: Helvetica, Arial, sans-serif;
  font-size: 12px;
  /* do not increase min-width as some may use split screens */
  min-width: 800px;
  color: #999;
}

h1 {
  font-size: 24px;
  color: black;
}

h2 {
  font-size: 16px;
  color: black;
}

p {
  color: black;
}

a {
  color: #999;
}

table {
  border-collapse: collapse;
}

/******************************
 * SUMMARY INFORMATION
 ******************************/
#environment td {
  padding: 5px;
  border: 1px solid #E6E6E6;
}
#environment tr:nth-child(odd) {
  background-color: #f6f6f6;
}

/******************************
 * TEST RESULT COLORS
 ******************************/
span.passed,
.passed .col-result {
  color: green;
}

span.skipped,
span.xfailed,
span.rerun,
.skipped .col-result,
.xfailed .col-result,
.rerun .col-result {
  color: orange;
}

span.error,
span.failed,
span.xpassed,
.error .col-result,
.failed .col-result,
.xpassed .col-result {
  color: red;
}

/******************************
 * RESULTS TABLE
 *
 * 1. Table Layout
 * 2. Extra
 * 3. Sorting items
 *
 ******************************/
/*------------------
 * 1. Table Layout
 *------------------*/
#results-table {
  border: 1px solid #e6e6e6;
  color: #999;
  font-size: 12px;
  width: 100%;
}
#results-table th,
#results-table td {
  padding: 5px;
  border: 1px solid #E6E6E6;
  text-align: left;
}
#results-table th {
  font-weight: bold;
}

/*------------------
 * 2. Extra
 *------------------*/
.log {
  background-color: #e6e6e6;
  border: 1px solid #e6e6e6;
  color: black;
  display: block;
  font-family: "Courier New", Courier, monospace;
  height: 230px;
  overflow-y: scroll;
  padding: 5px;
  white-space: pre-wrap;
}
.log:only-child {
  height: inherit;
}

div.image {
  border: 1px solid #e6e6e6;
  float: right;
  height: 240px;
  margin-left: 5px;
  overflow: hidden;
  width: 320px;
}
div.image img {
  width: 320px;
}

div.video {
  border: 1px solid #e6e6e6;
  float: right;
  height: 240px;
  margin-left: 5px;
  overflow: hidden;
  width: 320px;
}
div.video video {
  overflow: hidden;
  width: 320px;
  height: 240px;
}

.collapsed {
  display: none;
}

.expander::after {
  content: " (show details)";
  color: #BBB;
  font-style: italic;
  cursor: pointer;
}

.collapser::after {
  content: " (hide details)";
  color: #BBB;
  font-style: italic;
  cursor: pointer;
}

/*------------------
 * 3. Sorting items
 *------------------*/
.sortable {
  cursor: pointer;
}

.sort-icon {
  font-size: 0px;
  float: left;
  margin-right: 5px;
  margin-top: 5px;
  /*triangle*/
  width: 0;
  height: 0;
  border-left: 8px solid transparent;
  border-right: 8px solid transparent;
}
.inactive .sort-icon {
  /*finish triangle*/
  border-top: 8px solid #E6E6E6;
}
.asc.active .sort-icon {
  /*finish triangle*/
  border-bottom: 8px solid #999;
}
.desc.active .sort-icon {
  /*finish triangle*/
  border-top: 8px solid #999;
}
</style></head>
  <body onLoad="init()">
    <script>/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */


function toArray(iter) {
    if (iter === null) {
        return null;
    }
    return Array.prototype.slice.call(iter);
}

function find(selector, elem) { // eslint-disable-line no-redeclare
    if (!elem) {
        elem = document;
    }
    return elem.querySelector(selector);
}

function findAll(selector, elem) {
    if (!elem) {
        elem = document;
    }
    return toArray(elem.querySelectorAll(selector));
}

function sortColumn(elem) {
    toggleSortStates(elem);
    const colIndex = toArray(elem.parentNode.childNodes).indexOf(elem);
    let key;
    if (elem.classList.contains('result')) {
        key = keyResult;
    } else if (elem.classList.contains('links')) {
        key = keyLink;
    } else {
        key = keyAlpha;
    }
    sortTable(elem, key(colIndex));
}

function showAllExtras() { // eslint-disable-line no-unused-vars
    findAll('.col-result').forEach(showExtras);
}

function hideAllExtras() { // eslint-disable-line no-unused-vars
    findAll('.col-result').forEach(hideExtras);
}

function showExtras(colresultElem) {
    const extras = colresultElem.parentNode.nextElementSibling;
    const expandcollapse = colresultElem.firstElementChild;
    extras.classList.remove('collapsed');
    expandcollapse.classList.remove('expander');
    expandcollapse.classList.add('collapser');
}

function hideExtras(colresultElem) {
    const extras = colresultElem.parentNode.nextElementSibling;
    const expandcollapse = colresultElem.firstElementChild;
    extras.classList.add('collapsed');
    expandcollapse.classList.remove('collapser');
    expandcollapse.classList.add('expander');
}

function showFilters() {
    let visibleString = getQueryParameter('visible') || 'all';
    visibleString = visibleString.toLowerCase();
    const checkedItems = visibleString.split(',');

    const filterItems = document.getElementsByClassName('filter');
    for (let i = 0; i < filterItems.length; i++) {
        filterItems[i].hidden = false;

        if (visibleString != 'all') {
            filterItems[i].checked = checkedItems.includes(filterItems[i].getAttribute('data-test-result'));
            filterTable(filterItems[i]);
        }
    }
}

function addCollapse() {
    // Add links for show/hide all
    const resulttable = find('table#results-table');
    const showhideall = document.createElement('p');
    showhideall.innerHTML = '<a href="javascript:showAllExtras()">Show all details</a> / ' +
                            '<a href="javascript:hideAllExtras()">Hide all details</a>';
    resulttable.parentElement.insertBefore(showhideall, resulttable);

    // Add show/hide link to each result
    findAll('.col-result').forEach(function(elem) {
        const collapsed = getQueryParameter('collapsed') || 'Passed';
        const extras = elem.parentNode.nextElementSibling;
        const expandcollapse = document.createElement('span');
        if (extras.classList.contains('collapsed')) {
            expandcollapse.classList.add('expander');
        } else if (collapsed.includes(elem.innerHTML)) {
            extras.classList.add('collapsed');
            expandcollapse.classList.add('expander');
        } else {
            expandcollapse.classList.add('collapser');
        }
        elem.appendChild(expandcollapse);

        elem.addEventListener('click', function(event) {
            if (event.currentTarget.parentNode.nextElementSibling.classList.contains('collapsed')) {
                showExtras(event.currentTarget);
            } else {
                hideExtras(event.currentTarget);
            }
        });
    });
}

function getQueryParameter(name) {
    const match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
    return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
}

function init () { // eslint-disable-line no-unused-vars
    resetSortHeaders();

    addCollapse();

    showFilters();

    sortColumn(find('.initial-sort'));

    findAll('.sortable').forEach(function(elem) {
        elem.addEventListener('click',
            function() {
                sortColumn(elem);
            }, false);
    });
}

function sortTable(clicked, keyFunc) {
    const rows = findAll('.results-table-row');
    const reversed = !clicked.classList.contains('asc');
    const sortedRows = sort(rows, keyFunc, reversed);
    /* Whole table is removed here because browsers acts much slower
     * when appending existing elements.
     */
    const thead = document.getElementById('results-table-head');
    document.getElementById('results-table').remove();
    const parent = document.createElement('table');
    parent.id = 'results-table';
    parent.appendChild(thead);
    sortedRows.forEach(function(elem) {
        parent.appendChild(elem);
    });
    document.getElementsByTagName('BODY')[0].appendChild(parent);
}

function sort(items, keyFunc, reversed) {
    const sortArray = items.map(function(item, i) {
        return [keyFunc(item), i];
    });

    sortArray.sort(function(a, b) {
        const keyA = a[0];
        const keyB = b[0];

        if (keyA == keyB) return 0;

        if (reversed) {
            return keyA < keyB ? 1 : -1;
        } else {
            return keyA > keyB ? 1 : -1;
        }
    });

    return sortArray.map(function(item) {
        const index = item[1];
        return items[index];
    });
}

function keyAlpha(colIndex) {
    return function(elem) {
        return elem.childNodes[1].childNodes[colIndex].firstChild.data.toLowerCase();
    };
}

function keyLink(colIndex) {
    return function(elem) {
        const dataCell = elem.childNodes[1].childNodes[colIndex].firstChild;
        return dataCell == null ? '' : dataCell.innerText.toLowerCase();
    };
}

function keyResult(colIndex) {
    return function(elem) {
        const strings = ['Error', 'Failed', 'Rerun', 'XFailed', 'XPassed',
            'Skipped', 'Passed'];
        return strings.indexOf(elem.childNodes[1].childNodes[colIndex].firstChild.data);
    };
}

function resetSortHeaders() {
    findAll('.sort-icon').forEach(function(elem) {
        elem.parentNode.removeChild(elem);
    });
    findAll('.sortable').forEach(function(elem) {
        const icon = document.createElement('div');
        icon.className = 'sort-icon';
        icon.textContent = 'vvv';
        elem.insertBefore(icon, elem.firstChild);
        elem.classList.remove('desc', 'active');
        elem.classList.add('asc', 'inactive');
    });
}

function toggleSortStates(elem) {
    //if active, toggle between asc and desc
    if (elem.classList.contains('active')) {
        elem.classList.toggle('asc');
        elem.classList.toggle('desc');
    }

    //if inactive, reset all other functions and add ascending active
    if (elem.classList.contains('inactive')) {
        resetSortHeaders();
        elem.classList.remove('inactive');
        elem.classList.add('active');
    }
}

function isAllRowsHidden(value) {
    return value.hidden == false;
}

function filterTable(elem) { // eslint-disable-line no-unused-vars
    const outcomeAtt = 'data-test-result';
    const outcome = elem.getAttribute(outcomeAtt);
    const classOutcome = outcome + ' results-table-row';
    const outcomeRows = document.getElementsByClassName(classOutcome);

    for(let i = 0; i < outcomeRows.length; i++){
        outcomeRows[i].hidden = !elem.checked;
    }

    const rows = findAll('.results-table-row').filter(isAllRowsHidden);
    const allRowsHidden = rows.length == 0 ? true : false;
    const notFoundMessage = document.getElementById('not-found-message');
    notFoundMessage.hidden = !allRowsHidden;
}
</script>
    <h1>api-test-report-20250809-205712.html</h1>
    <p>Report generated on 09-Aug-2025 at 21:17:42 by <a href="https://pypi.python.org/pypi/pytest-html">pytest-html</a> v3.2.0</p>
    <h2>Summary</h2>
    <p>33 tests ran in 1229.17 seconds. </p>
    <p class="filter" hidden="true">(Un)check the boxes to filter the results.</p><input checked="true" class="filter" data-test-result="passed" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="passed">15 passed</span>, <input checked="true" class="filter" data-test-result="skipped" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="skipped">0 skipped</span>, <input checked="true" class="filter" data-test-result="failed" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="failed">18 failed</span>, <input checked="true" class="filter" data-test-result="error" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="error">0 errors</span>, <input checked="true" class="filter" data-test-result="xfailed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="xfailed">0 expected failures</span>, <input checked="true" class="filter" data-test-result="xpassed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="xpassed">0 unexpected passes</span>
    <h2>Results</h2>
    <table id="results-table">
      <thead id="results-table-head">
        <tr>
          <th class="sortable result initial-sort" col="result">Result</th>
          <th class="sortable" col="name">Test</th>
          <th class="sortable" col="duration">Duration</th>
          <th class="sortable links" col="links">Links</th></tr>
        <tr hidden="true" id="not-found-message">
          <th colspan="4">No results found. Try to check the filters</th></tr></thead>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestDirectorEndpoint::test_director_basic_request</td>
          <td class="col-duration">128.93</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   TimeoutError: The read operation timed out</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/director (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.ConnectionError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/director (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /director
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/director (Caused by 
ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.am
azonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))
<br/> -------------------------------Captured log call-------------------------------- <br/>WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestDirectorEndpoint::test_director_with_user_request</td>
          <td class="col-duration">129.91</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   TimeoutError: The read operation timed out</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/director?user_request=Test%20mission%20from%20Python%20API%20tests (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.ConnectionError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/director?user_request=Test%20mission%20from%20Python%20API%20tests (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /director?user_request=Test mission from Python API tests
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: 
/dev/director?user_request=Test%20mission%20from%20Python%20API%20tests (Caused
by 
ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.am
azonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))
<br/> -------------------------------Captured log call-------------------------------- <br/>WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director?user_request=Test%20mission%20from%20Python%20API%20tests
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director?user_request=Test%20mission%20from%20Python%20API%20tests
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director?user_request=Test%20mission%20from%20Python%20API%20tests<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestDirectorEndpoint::test_director_with_user_id</td>
          <td class="col-duration">130.13</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   TimeoutError: The read operation timed out</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/director?user_id=api-test-user (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.ConnectionError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/director?user_id=api-test-user (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /director?user_id=api-test-user
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/director?user_id=api-test-user 
(Caused by 
ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.am
azonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))
<br/> -------------------------------Captured log call-------------------------------- <br/>WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director?user_id=api-test-user
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director?user_id=api-test-user
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director?user_id=api-test-user<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestPersonaEndpoint::test_persona_valid_message</td>
          <td class="col-duration">8.57</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">urllib3.exceptions.ResponseError: too many 500 error responses<br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/persona (Caused by ResponseError(&#x27;too many 500 error responses&#x27;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.RetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/persona (Caused by ResponseError(&#x27;too many 500 error responses&#x27;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /persona
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/persona (Caused by 
ResponseError(&#x27;too many 500 error responses&#x27;))
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestElevatorEndpoint::test_elevator_call_basic</td>
          <td class="col-duration">129.28</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   TimeoutError: The read operation timed out</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/elevator/call (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.ConnectionError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/elevator/call (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /elevator/call
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/elevator/call (Caused by 
ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.am
azonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))
<br/> -------------------------------Captured log call-------------------------------- <br/>WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestElevatorEndpoint::test_elevator_missing_mission_id</td>
          <td class="col-duration">130.12</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   TimeoutError: The read operation timed out</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/elevator/call (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.ConnectionError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/elevator/call (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /elevator/call
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/elevator/call (Caused by 
ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.am
azonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))
<br/> -------------------------------Captured log call-------------------------------- <br/>WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestCoordinatorEndpoint::test_coordinator_mission_status</td>
          <td class="col-duration">2.52</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   assert 404 == 200</span><br/>     +  where 404 = &lt;Response [404]&gt;.status_code<br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /coordinator/missions/test-mission-123
📊 Status: 404 | ⏱️  Time: 2519.72ms | 📦 Size: 47 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestCoordinatorEndpoint::test_coordinator_different_mission_id</td>
          <td class="col-duration">0.25</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   assert 404 == 200</span><br/>     +  where 404 = &lt;Response [404]&gt;.status_code<br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /coordinator/missions/python-test-1754784499
📊 Status: 404 | ⏱️  Time: 254.44ms | 📦 Size: 53 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestCORSHeaders::test_cors_headers_present[/health]</td>
          <td class="col-duration">0.52</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   AssertionError: No CORS headers found in response for /health</span><br/>    assert False<br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /health
📊 Status: 200 | ⏱️  Time: 518.15ms | 📦 Size: 60 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestCORSHeaders::test_cors_headers_present[/director]</td>
          <td class="col-duration">129.01</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   TimeoutError: The read operation timed out</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/director (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.ConnectionError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/director (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /director
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/director (Caused by 
ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.am
azonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))
<br/> -------------------------------Captured log call-------------------------------- <br/>WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestCORSHeaders::test_cors_headers_present[/persona]</td>
          <td class="col-duration">1.90</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   AssertionError: No CORS headers found in response for /persona</span><br/>    assert False<br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /persona
📊 Status: 400 | ⏱️  Time: 1900.39ms | 📦 Size: 45 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestCORSHeaders::test_cors_headers_present[/elevator/call]</td>
          <td class="col-duration">129.30</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   TimeoutError: The read operation timed out</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/elevator/call (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.ConnectionError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/elevator/call (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /elevator/call
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/elevator/call (Caused by 
ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.am
azonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))
<br/> -------------------------------Captured log call-------------------------------- <br/>WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestCORSHeaders::test_cors_headers_present[/psim/search]</td>
          <td class="col-duration">1.27</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   AssertionError: No CORS headers found in response for /psim/search</span><br/>    assert False<br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /psim/search
📊 Status: 400 | ⏱️  Time: 1268.49ms | 📦 Size: 43 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_endpoints.py::TestCORSHeaders::test_cors_headers_present[/coordinator/missions/test-123]</td>
          <td class="col-duration">0.40</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   AssertionError: No CORS headers found in response for /coordinator/missions/test-123</span><br/>    assert False<br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /coordinator/missions/test-123
📊 Status: 404 | ⏱️  Time: 405.94ms | 📦 Size: 39 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_performance.py::TestPerformance::test_response_time_under_threshold</td>
          <td class="col-duration">129.54</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   TimeoutError: The read operation timed out</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/director (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.ConnectionError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/director (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /health
📊 Status: 200 | ⏱️  Time: 513.92ms | 📦 Size: 60 bytes

🚀 GET /director
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/director (Caused by 
ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.am
azonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))
<br/> -------------------------------Captured log call-------------------------------- <br/>WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/director<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_performance.py::TestStressScenarios::test_large_payload_persona</td>
          <td class="col-duration">7.15</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">urllib3.exceptions.ResponseError: too many 500 error responses<br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/persona (Caused by ResponseError(&#x27;too many 500 error responses&#x27;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.RetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/persona (Caused by ResponseError(&#x27;too many 500 error responses&#x27;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /persona
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/persona (Caused by 
ResponseError(&#x27;too many 500 error responses&#x27;))
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_performance.py::TestStressScenarios::test_special_characters_in_requests</td>
          <td class="col-duration">6.81</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">urllib3.exceptions.ResponseError: too many 500 error responses<br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/persona (Caused by ResponseError(&#x27;too many 500 error responses&#x27;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.RetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/persona (Caused by ResponseError(&#x27;too many 500 error responses&#x27;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /persona
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/persona (Caused by 
ResponseError(&#x27;too many 500 error responses&#x27;))
<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">test_performance.py::TestBoundaryConditions::test_empty_payloads</td>
          <td class="col-duration">129.33</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"><span class="error">E   TimeoutError: The read operation timed out</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)</span><br/><br/>The above exception was the direct cause of the following exception:<br/><span class="error">E   urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/elevator/call (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/><br/>During handling of the above exception, another exception occurred:<br/><span class="error">E   requests.exceptions.ConnectionError: HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Max retries exceeded with url: /dev/elevator/call (Caused by ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))</span><br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /persona
📊 Status: 400 | ⏱️  Time: 184.27ms | 📦 Size: 45 bytes

🚀 POST /elevator/call
❌ Request failed: 
HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, 
port=443): Max retries exceeded with url: /dev/elevator/call (Caused by 
ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.am
azonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;))
<br/> -------------------------------Captured log call-------------------------------- <br/>WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by &#x27;ReadTimeoutError(&quot;HTTPSConnectionPool(host=&#x27;pj4vlvxrg7.execute-api.us-east-1.amazonaws.com&#x27;, port=443): Read timed out. (read timeout=30)&quot;)&#x27;: /dev/elevator/call<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_endpoints.py::TestHealthEndpoint::test_health_check_success</td>
          <td class="col-duration">1.53</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> -----------------------------Captured stdout setup------------------------------ <br/>
🧪 Starting BuildingOS API Test Suite
🌐 Base URL: https://pj4vlvxrg7.execute-api.us-east-1.amazonaws.com/dev
🏷️  Environment: dev
<br/> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /health
📊 Status: 200 | ⏱️  Time: 1524.26ms | 📦 Size: 60 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_endpoints.py::TestHealthEndpoint::test_health_check_performance</td>
          <td class="col-duration">0.22</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /health
📊 Status: 200 | ⏱️  Time: 218.74ms | 📦 Size: 60 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_endpoints.py::TestPersonaEndpoint::test_persona_missing_user_id</td>
          <td class="col-duration">0.19</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /persona
📊 Status: 400 | ⏱️  Time: 193.72ms | 📦 Size: 45 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_endpoints.py::TestPersonaEndpoint::test_persona_missing_message</td>
          <td class="col-duration">0.19</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /persona
📊 Status: 400 | ⏱️  Time: 193.80ms | 📦 Size: 45 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_endpoints.py::TestPersonaEndpoint::test_persona_conversations</td>
          <td class="col-duration">0.23</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /persona/conversations?user_id=api-test-user
📊 Status: 200 | ⏱️  Time: 228.41ms | 📦 Size: 64 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_endpoints.py::TestPSIMEndpoint::test_psim_search_basic</td>
          <td class="col-duration">2.64</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /psim/search
📊 Status: 200 | ⏱️  Time: 2624.57ms | 📦 Size: 223 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_endpoints.py::TestPSIMEndpoint::test_psim_search_with_custom_query</td>
          <td class="col-duration">0.33</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /psim/search
📊 Status: 200 | ⏱️  Time: 327.13ms | 📦 Size: 223 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_endpoints.py::TestErrorHandling::test_invalid_endpoint</td>
          <td class="col-duration">0.19</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /invalid-endpoint
📊 Status: 404 | ⏱️  Time: 189.70ms | 📦 Size: 23 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_endpoints.py::TestErrorHandling::test_invalid_method</td>
          <td class="col-duration">0.19</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 DELETE /health
📊 Status: 404 | ⏱️  Time: 188.80ms | 📦 Size: 23 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_endpoints.py::TestErrorHandling::test_malformed_json</td>
          <td class="col-duration">7.52</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ----------------------------Captured stdout teardown---------------------------- <br/>
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_performance.py::TestPerformance::test_concurrent_health_checks</td>
          <td class="col-duration">2.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /health

🚀 GET /health

🚀 GET /health

🚀 GET /health

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 1904.54ms | 📦 Size: 60 bytes
📊 Status: 200 | ⏱️  Time: 1917.21ms | 📦 Size: 60 bytes
📊 Status: 200 | ⏱️  Time: 1950.75ms | 📦 Size: 60 bytes
📊 Status: 200 | ⏱️  Time: 1998.10ms | 📦 Size: 60 bytes
📊 Status: 200 | ⏱️  Time: 1998.06ms | 📦 Size: 60 bytes
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_performance.py::TestLoadTesting::test_sustained_load_health_endpoint</td>
          <td class="col-duration">13.84</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🔥 Running sustained load test...

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 594.46ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 187.34ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 180.01ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 197.85ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 202.39ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 214.14ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 196.35ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 193.30ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 196.68ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 197.87ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 190.84ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 187.19ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 212.46ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 182.19ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 183.73ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 204.71ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 202.21ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 179.45ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 197.94ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 187.06ms | 📦 Size: 60 bytes
  Making 20 requests... 0:00:13
✅ Success rate: 100.0% (20/20)
⏱️  Avg response: 216.38ms
🐌 Max response: 596.12ms
⚡ Min response: 181.16ms
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_performance.py::TestStressScenarios::test_rapid_successive_requests</td>
          <td class="col-duration">1.94</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET /health
📊 Status: 200 | ⏱️  Time: 202.03ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 184.31ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 188.70ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 191.75ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 215.73ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 188.91ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 188.07ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 203.35ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 178.11ms | 📦 Size: 60 bytes

🚀 GET /health
📊 Status: 200 | ⏱️  Time: 191.29ms | 📦 Size: 60 bytes
🚀 Rapid requests: 10 in 1.95s
✅ Success rate: 100.0%
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_performance.py::TestBoundaryConditions::test_null_values_in_payload</td>
          <td class="col-duration">1.17</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 POST /persona
📊 Status: 400 | ⏱️  Time: 1175.99ms | 📦 Size: 45 bytes
<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_performance.py::TestBoundaryConditions::test_extremely_long_mission_id</td>
          <td class="col-duration">0.45</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stdout call------------------------------ <br/>
🚀 GET 
/coordinator/missions/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
📊 Status: 404 | ⏱️  Time: 320.10ms | 📦 Size: 1031 bytes
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>
📊 Test Session Complete
                              🔍 Request Summary                               
┌────────┬─────────────────────────────────────┬────────┬───────────┬─────────┐
│ Method │ Endpoint                            │ Status │ Time (ms) │ Success │
├────────┼─────────────────────────────────────┼────────┼───────────┼─────────┤
│ GET    │ /health                             │ 200    │ 1524.26   │ ✅      │
│ GET    │ /health                             │ 200    │ 218.74    │ ✅      │
│ GET    │ /director                           │ 0      │ 128926.82 │ ❌      │
│ GET    │ /director?user_request=Test mission │ 0      │ 129913.69 │ ❌      │
│        │ from Python API tests               │        │           │         │
│ GET    │ /director?user_id=api-test-user     │ 0      │ 130120.5  │ ❌      │
│ POST   │ /persona                            │ 0      │ 8568.7    │ ❌      │
│ POST   │ /persona                            │ 400    │ 193.72    │ ❌      │
│ POST   │ /persona                            │ 400    │ 193.8     │ ❌      │
│ GET    │ /persona/conversations?user_id=api… │ 200    │ 228.41    │ ✅      │
│ POST   │ /elevator/call                      │ 0      │ 129284.5  │ ❌      │
│ POST   │ /elevator/call                      │ 0      │ 130117.21 │ ❌      │
│ POST   │ /psim/search                        │ 200    │ 2624.57   │ ✅      │
│ POST   │ /psim/search                        │ 200    │ 327.13    │ ✅      │
│ GET    │ /coordinator/missions/test-mission… │ 404    │ 2519.72   │ ❌      │
│ GET    │ /coordinator/missions/python-test-… │ 404    │ 254.44    │ ❌      │
│ GET    │ /health                             │ 200    │ 518.15    │ ✅      │
│ GET    │ /director                           │ 0      │ 129015.1  │ ❌      │
│ POST   │ /persona                            │ 400    │ 1900.39   │ ❌      │
│ POST   │ /elevator/call                      │ 0      │ 129295.91 │ ❌      │
│ POST   │ /psim/search                        │ 400    │ 1268.49   │ ❌      │
│ GET    │ /coordinator/missions/test-123      │ 404    │ 405.94    │ ❌      │
│ GET    │ /invalid-endpoint                   │ 404    │ 189.7     │ ❌      │
│ DELETE │ /health                             │ 404    │ 188.8     │ ❌      │
│ GET    │ /health                             │ 200    │ 513.92    │ ✅      │
│ GET    │ /director                           │ 0      │ 129025.1  │ ❌      │
│ GET    │ /health                             │ 200    │ 1904.54   │ ✅      │
│ GET    │ /health                             │ 200    │ 1917.21   │ ✅      │
│ GET    │ /health                             │ 200    │ 1950.75   │ ✅      │
│ GET    │ /health                             │ 200    │ 1998.1    │ ✅      │
│ GET    │ /health                             │ 200    │ 1998.06   │ ✅      │
│ GET    │ /health                             │ 200    │ 594.46    │ ✅      │
│ GET    │ /health                             │ 200    │ 187.34    │ ✅      │
│ GET    │ /health                             │ 200    │ 180.01    │ ✅      │
│ GET    │ /health                             │ 200    │ 197.85    │ ✅      │
│ GET    │ /health                             │ 200    │ 202.39    │ ✅      │
│ GET    │ /health                             │ 200    │ 214.14    │ ✅      │
│ GET    │ /health                             │ 200    │ 196.35    │ ✅      │
│ GET    │ /health                             │ 200    │ 193.3     │ ✅      │
│ GET    │ /health                             │ 200    │ 196.68    │ ✅      │
│ GET    │ /health                             │ 200    │ 197.87    │ ✅      │
│ GET    │ /health                             │ 200    │ 190.84    │ ✅      │
│ GET    │ /health                             │ 200    │ 187.19    │ ✅      │
│ GET    │ /health                             │ 200    │ 212.46    │ ✅      │
│ GET    │ /health                             │ 200    │ 182.19    │ ✅      │
│ GET    │ /health                             │ 200    │ 183.73    │ ✅      │
│ GET    │ /health                             │ 200    │ 204.71    │ ✅      │
│ GET    │ /health                             │ 200    │ 202.21    │ ✅      │
│ GET    │ /health                             │ 200    │ 179.45    │ ✅      │
│ GET    │ /health                             │ 200    │ 197.94    │ ✅      │
│ GET    │ /health                             │ 200    │ 187.06    │ ✅      │
│ POST   │ /persona                            │ 0      │ 7148.02   │ ❌      │
│ GET    │ /health                             │ 200    │ 202.03    │ ✅      │
│ GET    │ /health                             │ 200    │ 184.31    │ ✅      │
│ GET    │ /health                             │ 200    │ 188.7     │ ✅      │
│ GET    │ /health                             │ 200    │ 191.75    │ ✅      │
│ GET    │ /health                             │ 200    │ 215.73    │ ✅      │
│ GET    │ /health                             │ 200    │ 188.91    │ ✅      │
│ GET    │ /health                             │ 200    │ 188.07    │ ✅      │
│ GET    │ /health                             │ 200    │ 203.35    │ ✅      │
│ GET    │ /health                             │ 200    │ 178.11    │ ✅      │
│ GET    │ /health                             │ 200    │ 191.29    │ ✅      │
│ POST   │ /persona                            │ 0      │ 6812.14   │ ❌      │
│ POST   │ /persona                            │ 400    │ 184.27    │ ❌      │
│ POST   │ /elevator/call                      │ 0      │ 129147.79 │ ❌      │
│ POST   │ /persona                            │ 400    │ 1175.99   │ ❌      │
│ GET    │ /coordinator/missions/xxxxxxxxxxxx… │ 404    │ 320.1     │ ❌      │
└────────┴─────────────────────────────────────┴────────┴───────────┴─────────┘
┌──────────────────────────── Performance Metrics ────────────────────────────┐
│ 📈 Avg: 522.48ms | ⚡ Min: 178.11ms | 🐌 Max: 2624.57ms                     │
└─────────────────────────────────────────────────────────────────────────────┘
💾 Results exported to: api-test-results-20250809-211742.json
<br/></div></td></tr></tbody></table></body></html>